articles/iot-accelerators/iot-accelerators-connecting-pi-node.md

```diff
@@ -58,7 +58,7 @@ If you haven't done so already, install Node.js version 4.0.0 or later on your R
 
     ```sh
     curl -sL https://deb.nodesource.com/setup_6.x | sudo bash -
-    sudo apt-get install nodejs -y
+    sudo apt-get install nodejs npm node-semver
```

Comment:
I don't think you need to include node-semver - it's installed automatically as dependency of npm

Reply:
Sounds good. Will remove.